kernels/TIndexKernel.hpp

```diff
@@ -34,9 +34,17 @@
 
 #pragma once
 
+#include <pdal/Filter.hpp>
+#include <pdal/Streamable.hpp>
 #include <pdal/Stage.hpp>
 #include <pdal/SubcommandKernel.hpp>
 #include <pdal/util/FileUtils.hpp>
+#include <filters/private/hexer/HexGrid.hpp>
+
+namespace hexer
```

Comment:
why do we need this forward declaration if we're also including the `HexGrid.hpp` header?